src/main/java/net/snowflake/client/jdbc/SnowflakeChunkDownloader.java

```diff
@@ -306,9 +307,18 @@ private void startNextDownloaders() throws SnowflakeSQLException {
 
       // try to reserve the needed memory
       long curMem = currentMemoryUsage.addAndGet(neededChunkMemory);
+      logger.debug(
```

Comment:
Is this new log line necessary? It seems some info will be printed out again (see line 299)